logDeployment.ps1

```diff
@@ -6,7 +6,9 @@ param(
     [parameter(Mandatory = $true)]
     [string]$env,
     [parameter(Mandatory = $true)]
-    [string]$vendorKey
+    [string]$vendorKey,
+    [parameter(Mandatory = $false)]
+    [DateTimeOffset]$deploymentDateTimeOffset
```

Comment:
Would you be able to add comments or a docstring to describe what these params are?

Reply:
added